seven/Counter.java

```diff
@@ -0,0 +1,25 @@
+package seven;
+
+import java.util.concurrent.TimeUnit;
+public class Counter {
+ void printCount()
+ {
+	 for(int i=1;i<=100;i++)
+	 {  
+		 try
+		 {  
+			TimeUnit.MILLISECONDS.sleep(1000);
```

Comment:
@Hariharan-s005 why TimeUnit.MILLISECONDS explicitly, however by default it would be ms? (And what is the important's of thread sleep() here?)

@jenisha-roshan